dependency-check-suppressions.xml

```diff
@@ -30,10 +30,20 @@
         <cve>CVE-2019-17632</cve>
     </suppress>
 
+    <suppress>
+        <cve>CVE-2020-9488</cve>
+        <cve>CWE-79</cve>
```

Comment:
this was invalid format and failed parsing, fixed in https://github.com/hmcts/finrem-case-orchestration-service/pull/315/commits/4457b0a03dab00500899a93754e48e050ab00000